src/core/reducers/errorPage.js

```diff
@@ -0,0 +1,42 @@
+import {
+  REDUX_CONNECT_END_GLOBAL_LOAD,
+  REDUX_CONNECT_LOAD_FAIL,
+} from 'core/constants';
+
+export const initialState = {
+  clearOnNext: false,
+  error: null,
+  hasError: false,
+  statusCode: null,
+};
+
+export default function errorPage(state = initialState, action) {
+  const { payload } = action;
+  switch (action.type) {
+    case REDUX_CONNECT_END_GLOBAL_LOAD:
```

Comment:
I believe I tried that and the issue is that then it would clear error state and try to render the non-error-page content and error. I'll double-check, but I think I needed to use `END_GLOBAL_LOAD` to prevent errors from throwing.